app/mailers/admin_mailer.rb

```diff
@@ -78,4 +80,10 @@ def affiliates_registration
 
     mail(to: recipient.email, subject: @notification.title)
   end
+
+  def linkedin_weekly_profiles
+    @developer_external_profiles = params[:developer_external_profiles_records]
+    subject = "#{pluralize(@developer_external_profiles&.count, "developer")} updated their LinkedIn profile in the last 7 days"
```

Comment:
1. Let's rename this to `linkedin_profiles`.
2. We can use [`String#pluralize`](https://apidock.com/rails/String/pluralize) and not delegate to the Rails helper.
3. Why are we optional chaining here? Shouldn't this always be present?

Reply:
yes there is no need to optional chaining here. Have updated code and made other changes as well.